### PR TITLE
cleanup: update files

### PR DESCRIPTION
**Pull request by kbdharun**

### CODE REVIEW

**Review by sebastiaanspeck — APPROVED** [review]

File: LICENSE.txt
Comment (sebastiaanspeck) [inline LICENSE.txt:1]: Why not .md?

File: LICENSE.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2017 tldr pages
+Copyright (c) 2017 - present tldr pages contributors
```
Comment (sebastiaanspeck) [inline LICENSE.txt:3]:
```suggestion
Copyright © 2017—present the [tldr-pages team](https://github.com/orgs/tldr-pages/people)

File: LICENSE.txt
Comment (kbdharun) [reply LICENSE.txt:1]: Many of the License files generally aren't structured properly in Markdown so I generally use TXT files. In this case, anything works.

File: LICENSE.txt (hunk 1 above)
Comment (kbdharun) [reply LICENSE.txt:3]: Feel free to add this in a commit.

**Pull request merged**